vulcan/_message.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime
+
+from related import (
+    IntegerField,
+    StringField,
+    DateField,
+    ChildField,
+    immutable,
+    to_model,
+)
+
+from ._subject import Subject
+from ._teacher import Teacher
+from ._utils import sort_and_filter_date
+@immutable
+class Message:
+	sender = StringField(key="Nadawca")
+	senderid = StringField(key="NadawcaId")
+	addressees = StringField(key="Adresaci")
+	title = StringField(key="Tytul")
+	content = StringField(key="Tresc")
+	postdate = StringField(key="DataWyslania")
+	posthour = StringField(key="GodzinaWyslania")
+	folder= StringField(key="FolderWiadomosci")
+	"""
+	messageid = ChildField(MessageId, required=False)
+	"""
+	senderid = ChildField(Teacher, required=False)
+	@classmethod
+	def get(cls, api, date=None):
+		if not date:
+			date = datetime.now()
+		date_str = date.strftime("%Y-%m-%d")
+
+		date = {"DataPoczatkowa": "2020-01-01", "DataKoncowa": date_str}
```

Comment:
Do **NOT** use static dates! It would be better to remove `date` argument from this method and use period start and end dates (`api.student.period.from_` and `api.student.period.to`, don't forget to `strftime` this)
And by the way, change `date` var name to `data`. Don't forget to change it in the next line too.

Reply:
ooo
I forgot about this.